pkg/networkserver/downlink.go

```diff
@@ -849,14 +846,26 @@ loop:
 	return phy.DataRates[maxUpDRIdx].DefaultMaxSize.PayloadSize(fp.DwellTime.GetUplinks())
 }
 
+// networkInitiatedDownlinkInterval is the minimum time.Duration passed before a network-initiated(e.g. Class B or C) downlink following an arbitrary downlink.
+const networkInitiatedDownlinkInterval = 5 * time.Second
```

Comment:
What is this 5 seconds based on? Is this a trade-off?

Reply:
Arbitrary value - this is also the default RxDelay we use and Join-request delay, so if a device would send an uplink we'd know about it by that time instead of trying to schedule another class B/C downlink.
What this value means is that if NS has `n` unconfirmed downlinks to schedule, it will schedule first one at `T`, second one at `T+networkInitiatedDownlinkInterval`, third one at `T+2*networkInitiatedDownlinkInterval` .. `n` at `T+(n-1)*networkInitiatedDownlinkInterval`.

Reply:
It should probably be configurable on NS level at level at least, maybe even device

Reply:
Does this mean that in case of FOTA for example, many class C unconfirmed downlink messages, there's always 5 seconds in between?

Reply:
Yes, unless there's an uplink.
Do you think it should be lower?

Reply:
Yeah I think it should be lower indeed. Why don't you use the GS's delay on the schedule response?

Reply:
NS does that, but it still needs to have some kind of time interval to wait after the transmission by GS. To account for e.g. possible uplink.

Reply:
Lowered it to 1 second

Reply:
If it's unconfirmed downlink, I wouldn't bother too much about that.

Indeed we shouldn't send confirmed downlink too fast.